server.js

```diff
@@ -1,22 +1,63 @@
+if (process.env.NODE_ENV !== 'production') {
+  require('dotenv').config()
+};
+
 // Dependences
 const express = require("express");
+
+const session = require('express-session');
+const passport = require('passport');
+const flash = require('express-flash');
+const db = require('./models');
+const sequelize = require('sequelize');
+
 const messaging = require("./controllers/messaging/messagingServer");
 
+
 // Express config
 const app = express();
-const PORT = process.env.PORT || 3001;
+const PORT = process.env.PORT || 7001;
+
+// Setup Handlebars
+const exphbs = require("express-handlebars");
+
+app.engine("handlebars", exphbs({ defaultLayout: "main" }));
+app.set('view-engine', 'handlebars');
 
 // Express data handler config
 app.use(express.static("public"));
-app.use(express.urlencoded({ extended: true }));
+app.use(express.urlencoded({ extended: false }));
```

Comment:
Any reason to set this to false.

Reply:
yes that was intentially during dev - i think I can change it tho - i'll test prior to merge